Gems/Atom/RHI/Code/Include/Atom/RHI/Factory.h

```diff
@@ -93,6 +98,25 @@ namespace AZ
             /// Access the global factory instance.
             static Factory& Get();
 
+#if defined(USE_RENDERDOC)
+#if defined(AZ_PLATFORM_WINDOWS)
+            static const char* RENDERDOC_MODULE = "renderdoc.dll";
+#elif defined(AZ_PLATFORM_LINUX)
+            static const char* RENDERDOC_MODULE = "librenderdoc.so";
+#elif defined(AZ_PLATFORM_ANDROID)
+            static const char* RENDERDOC_MODULE = "libVkLayer_GLES_RenderDoc.so"
+#else
+            static const char* RENDERDOC_MODULE = nullptr;
+#endif
```

Comment:
This is usually done with platform traits rather than an ifdef block

Reply:
Gotcha, which files would they go in? This would block a merge since it seems like the validator blocks this

Reply:
Looks like there's no RHI platform traits yet. I'll have to add that